src/Microsoft.ML.ImageAnalytics/ImageType.cs

```diff
@@ -2,23 +2,72 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Drawing;
 using Microsoft.ML.Data;
 using Microsoft.ML.Internal.Utilities;
 using Microsoft.ML.Runtime;
 
 namespace Microsoft.ML.Transforms.Image
 {
+    /// <summary>
+    /// Allows a member to be marked as a <see cref="VectorDataViewType"/>, primarily allowing one to set
+    /// the dimensionality of the resulting array.
```

Comment:
` resulting array` what resulting array?

Reply:
Sorry for making copy-and-paste..

---
In reply to: [286693530](https://github.com/dotnet/machinelearning/pull/3263#discussion_r286693530) [](ancestors = 286693530)